alpha/lib/model/entry.php

```diff
@@ -363,11 +363,14 @@ public function setStatusReady ( $force = false )
 	 */
 	public function setModerationStatus($v)
```

Comment:
how you decide whether we always need to increase the moderation count if we got status ENTRY_MODERATION_STATUS_PENDING_MODERATION/ENTRY_MODERATION_STATUS_FLAGGED_FOR_REVIEW is it true on all cases?